### DISCUSSION TITLE
【有奖征文】ByConity云上快速部署体验有奖召集令

### DISCUSSION THREAD

**Discussion by serenallttyy** [ByConity/ByConity#623] [General]
引言：
6月初我们分享了ByConity在云上部署的方法：《[ByConity如何在Kubernetes上无感扩缩容](https://mp.weixin.qq.com/s/hi1Yh-tBBcGCyujD54-fyQ)》，之后收到很多用户反馈和建议，例如部署过程中需要安装的组件过多、本地软硬件环境引起部署失败等问题。因此我们做了一些优化，提供了更加容易上手的一键部署方式，邀请大家一起参与体验“ByConity 快速部署有奖征集活动”，并为大家准备了大奖！

多种云上快速部署
ByConity支持模块化和容器化的部署，可以直接部署在Kubernetes集群上，并且利用Kubernetes的弹性伸缩能力进行扩缩容。ByConity不仅可以在火山云上部署，也可以在阿里云、AWS、腾讯云等多种云上部署。通过视频介绍下ByConity如何快速在云上部署和扩缩容。
- 硬件配置：
用户需要购买和部署自己的Kubernetes集群，建议在不影响性能前提下，测试环境的硬件配置，以10G数据TPCDS测试的如下表1参考配置，其中Foundation DB的配置可以参考官方文档；生产环境建议表2的硬件配置，但也需要根据数据量和查询要求自由调节配置。
![image](https://github.com/ByConity/ByConity/assets/88301094/a818302d-6e28-4344-b5d1-0eea665d0b49)
表1 测试环境硬件配置建议   
![image](https://github.com/ByConity/ByConity/assets/88301094/a97f5ce8-947b-4896-aba4-8506fa5151da)
   表2 生产环境硬件配置建议   
- 软件安装：
  - 本地安装Kubernetes命令行工具kubectl，用于管理Kubernetes集群
  - 本地安装用于管理Kubernetes应用程序的包管理工具helm
  - 本地安装byconity-deploy代码：
![image](https://github.com/ByConity/ByConity/assets/88301094/4874e11c-7abf-4643-99cc-0188f16a2e90)

- 手把手视频教程（点击下方链接或添加小助手微信发送视频教程）
[https://bytedance.feishu.cn/file/MyEAbE1kVoBujvxsfQ8czrc1nQd?from=from_copylink](url)
视频1 ByConity在ECS集群上快速部署
ByConity提供了完整的部署文档和集群配置选项，详细可参考：https://byconity.github.io/zh-cn/docs/deployment/deploy-k8s。
支持场景介绍
ByConity 使用大量成熟 OLAP 技术，例如列存引擎，MPP 执行，智能查询优化，向量化执行，Codegen, Indexing，数据压缩，主要用于 OLAP 查询和计算场景。在实时数据接入、大宽表聚合查询、海量数据下复杂分析计算、多表关联查询场景下有非常好的性能，比较适合如下场景：
![image](https://github.com/ByConity/ByConity/assets/88301094/bf2cb4d2-649d-4f5b-a140-01dd3a46e22f)

表 3 ByConity支持场景分类
活动规则和要求
1. 使用提供的快速部署方式，在Kubenetes集群上完成 ByConity 的安装和部署
2. 完成后产出相关测试文章，并发布在第三方社区博客，如 InfoQ、掘金、CSDN、知乎等
  1. 文章需包括操作步骤&整体流程，对部署工具的使用感受&评价
  2. 参与团队&个人公开透明，GitHub 同步信息

活动奖品
名额：共 15 位
- 前10位符合要求且文章阅读量>100 的参与者，每人可获得 1000 元京东卡一张
- 5 位符合要求的参与者，每人可获得 500元京东卡一张
- 文章阅读量统计时间以活动截止日期为准
- 数量有限，先到先得

活动时间及参与方式
- 截止时间：9 月 30 日
- 参与方式：博客发布后同步链接在 [GitHub](https://github.com/ByConity/ByConity/discussions) 即表示参与活动（公司团队+文章链接），报名请联系ByConity 小助手
![image](https://github.com/ByConity/ByConity/assets/88301094/16e77b5e-8bd9-4f36-afc2-9eccd8979c27)

**Comment by 061501** [comment]
https://getup.blog.csdn.net/article/details/133191377

**Reply by 061501** [reply]
没人吗
